client/types/things.go

```diff
@@ -410,14 +422,21 @@ func (uf *UserFile) JSONMetadata() (json.RawMessage, error) {
 type WireSearchLibrary struct {
 	ThingHeader
 	SearchLibrary
-	Updated time.Time
+	WritableFields EnabledFields
+	Updated        time.Time
+}
+
+type EnabledFields struct {
+	Details bool
+	Sharing bool
```

Comment:
It would be nice to get info on if the current user can delete the thing, too. [Mentioned yesterday on the issue](https://github.com/gravwell/backend-issues/issues/5500#issuecomment-1928088171).